cmd/main.go

```diff
@@ -111,8 +113,8 @@ func main() {
 		version.NewVersionCmd(),
 	)
 
-	foundProject, version := getProjectVersion()
-	if foundProject && version == "1" {
+	foundProject, versionString := getProjectVersion()
+	if foundProject && versionString == "1" {
```

Comment:
I do not think that is a good practice to add the type of var in the nomenclature of it.

Reply:
`version` is a package, what do you suggest?

Reply:
My suggestion would be:

```suggestion
	if isProjectFound && projectVersion == "1" {
```
WDYT? Note that I'd use here the prefix is because is a boolean. (the nomenclature to make clear the understanding for boolean is to use `is` or `has` as a prefix)

Also, note that I'd apply the same above recommendation over it as well. I understand that your motivation here is to improve the understanding of the code in this func. So, it shows enough for we do a PR just with it.

Reply:
Hi @Adirio, 

After your comment, my recommendation/suggestion would be hasProjectFile since it is a boolean. 

Just to share. But, it is a really tiny thing so. I am ok with how it is today. 

Note that: `Booleans can hold only 2 values, true or false. Given this, using prefixes like “is”, “has”, and “can” will help the reader infer the type of the variable.` From [ref](https://hackernoon.com/the-art-of-naming-variables-52f44de00aad). However, if you looking for it you will see that in order to follow the nomenclature good practices for booleans usually, we should use `is` or `has` as a prefix.  See more few refs [1](https://www.samanthaming.com/tidbits/34-better-boolean-variable-names) and [2](http://bouncetadiss.blogspot.com/2012/04/good-practice-when-naming-boolean.html)